SwrveDemo/Podfile

```diff
@@ -0,0 +1,7 @@
+
+target 'SwrveDemoFramework' do
+    pod 'SwrveSDKCommon', :path => '../'
+    pod 'SwrveConversationSDK', :path => '../'
+    pod 'SwrveSDK', :path => '../'
+    #pod 'SwrveSDK', '~> 4.5.0'
```

Comment:
Perhaps add a comment to above line, explaining to comment out above 3 lines and uncomment below to use remote cocoapod....